src/http-context.ts

```diff
@@ -48,7 +48,7 @@ export class HttpContext {
     }
 
     const { headers, header, url: rawUrl, method: m } = req.json();
-    const method = m ?? 'GET';
+    const method = m?.toUpperCase() ?? 'GET';
```

Comment:
i noticed there are explicit conditional checks for uppercase below. is there somewhere else this could be getting set?

Reply:
Nope – I think this code used to assume that the pdk will always call with uppercased HTTP verbs.